WPMedia/arc-themes-components
# THEMES-1066: Re-enable Storybook for components

## Description

#### Jira Ticket: [THEMES-1066](https://arcpublishing.atlassian.net/browse/THEMES-1066)

This PR updates the Storybook files to work with v8 of the framework. Other updates added to make sure checks will pass in Github for PRs.

## Test Steps

1. Checkout this branch - `git checkout THEMES-1066`
2. Update dependencies - `npm i`
3. Run Storybook `npm run storybook`
4. Verify that the doc pages and stories load.


[THEMES-1066]: https://arcpublishing.atlassian.net/browse/THEMES-1066?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
